framework/src/org/apache/cordova/CordovaInterfaceImpl.java

```diff
@@ -147,6 +151,8 @@ public void onSaveInstanceState(Bundle outState) {
             String serviceName = activityResultCallback.getServiceName();
             outState.putString("callbackService", serviceName);
         }
+
+        outState.putBundle("plugin", pluginManager.onSaveInstanceState());
```

Comment:
pluginManager should be created when the first URL is loaded. In the default Android project, this happens in `onCreate()` so it shouldn't be an issue.